### PR TITLE
Remove HazardSite.location field

### PR DESCRIPTION
**Pull request by micheles**
See https://bugs.launchpad.net/oq-engine/+bug/1473051. The tests are running here: https://ci.openquake.org/job/zdevel_oq-engine/1338.

### CODE REVIEW

File: qa_tests/hazard/event_based/sc_utils.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def joint_prob_of_occurrence(gmvs_site_1, gmvs_site_2, gmv, time_span,
         if gmv_close(gmv_site_1) and gmv_close(gmv_site_2):
             count += 1
 
-    prob = 1 - math.exp(- (float(count) / (time_span * num_ses)) * time_span)
```
Comment (daniviga) [inline qa_tests/hazard/event_based/sc_utils.py]: Why this change? Is it related to this PR?

Comment (micheles) [inline qa_tests/hazard/event_based/sc_utils.py]: It is not related. It is not even a change:

``count / (time_span * num_ses) * time_span == count / num_ses``

**Comment by daniviga** [comment]
LGTM
